packages/extension-api/src/extension-api.d.ts

```diff
@@ -3193,5 +3193,13 @@ declare module '@podman-desktop/api' {
 
     // Navigate to a specific contribution (aka extension page) referenced by name
```

Comment:
Not related to this PR but we might want to give more informations on _what's the difference between contributions and webview_